src/styles/blocks/icons.scss

```diff
@@ -0,0 +1,33 @@
+
+.icon {
+  display: flex;
+  vertical-align: middle;
+  height: 24px;
+  width: 24px;
+  background-image: url(../images/icons/icon-burger-menu.svg);
+  background-size: cover;
+  transition: transform 0.3s;
+
+  @include hover(transform, scale(1.2));
```

Comment:
Replace the @include hover(transform, scale(1.2)); with &:hover { transform: scale(1.2); } for better readability.